packages/dependency-extraction-webpack-plugin/lib/index.js

```diff
@@ -64,9 +120,14 @@ class DependencyExtractionWebpackPlugin {
 			return callback( null, { this: externalRequest } );
 		}
 
+		// @ts-ignore
 		return callback();
```

Comment:
This should be a valid callback according to docs. I've created a PR in DefinitelyTyped that fixes it:

https://github.com/DefinitelyTyped/DefinitelyTyped/pull/45077